packages/dataviews/src/dataviews-layouts/grid/style.scss

```diff
@@ -7,10 +7,10 @@
 	padding: 0 $grid-unit-60 $grid-unit-30;
 	container-type: inline-size;
 	/**
-	 * Breakpoints were adjusted from media queries breakpoints to account for
-	 * the sidebar width. This was done to match the existing styles we had.
+	 * Match the padding applied to other DataViews components at the same
+	 * max-width.
 	 */
-	@container (max-width: 480px) {
+	@container (max-width: 430px) {
```

Comment:
Oh interesting, I wonder why the discrepancy was introduced? I believe the 480px value was reached from some calculation based on the gutenberg media queries with the site editor sidebar width. I can't find any explanation for where the 430 value comes from but I see it's also used in `edit-site`.